cv32e40p/docs/VerifPlans/README.md

```diff
@@ -1,15 +1,22 @@
-<!--- SPDX-License-Identifier: Apache-2.0 WITH SHL-2.0 ---> 
+<!--- SPDX-License-Identifier: Apache-2.0 WITH SHL-2.0 --->
 This is the root directory of the CV32E40P Verification Plan (aka Test Plan).  Each sub-directory is the Verification Plan a specific CV32E40P high-level feature of the same name.
 
 Use the provided CORE-V_Simulation VerifPlan_Template.xlsx spreadsheet as your template to capture a Verification Plan.
 
+Below are two different chapters describing verification plans status and direct link for CV32E40P**v2** verification plans first, and CV32E40P**v1** after.
+
+# CV32E40P (V2) Verification Plans
+
+## Short verification methodology introduction
+For CV32E40P**v2** verification, the formal verification methodology has been choosen over the stimuli-based simulation that was done for version one of the core. However, full verification closure is not feasible using only formal verification due to complexity of specific scenarios. All these specific uncoverable scenarios from formal verification are then exercised by stimuli-based simulation using a reference model of the core. These scenarios along with formal assertions are described inside verifications plans, for which details are given in a table below. Regarding already available v1 plans, their re-use or not is specified in this table.
+
 ## Verification Plan Status
 
-The tables below capture the current status of the Verification Plan for the CV32E40P by high-level feature, as long with status update with respect to CV32E40Pv1 verification plans.  Under the heading `Capture`, the test plan can be **Incomplete**, **draft**, or one of the following: 
-* **Captured _(v1)_**: Vplan has been captured during v1 verification and has no modifications since then. 
-* **Captured _(new)_**: Vplan captures either new features or features not tested in v1
-* **Captured _(formal)_**: Vplan captures a different methodology from v1, using formal tool
-* **Captured _(updated)_**: Vplan has been captured in v1, but contains modifications for v2
+The tables below capture the current status of the Verification Plan for the CV32E40P by high-level feature, as long with status update with respect to CV32E40Pv1 verification plans.  Under the heading `Capture`, the test plan can be **Incomplete**, **draft**. If the verification is **captured**, it has one of the following status:
```

Comment:
The heading `Capture` is used in the V1 tables, but in the V2 table the heading is `VPlan Status`.  Maybe we can use VPlan Status for both.

Reply:
There is a typo in the part of the file you are showing, I meant to use `VPlan Status` as a header and not `Capture`, I forgot to change that. For the V1 table, the idea was to left the v1 work as it was before starting working on the v2, but I can change little things like this if you prefer